contracts/TopShot.cdc

```diff
@@ -47,62 +47,65 @@ import FungibleToken from 0xFUNGIBLETOKENADDRESS
 import NonFungibleToken from 0xNFTADDRESS
 import MetadataViews from 0xMETADATAVIEWSADDRESS
 import TopShotLocking from 0xTOPSHOTLOCKINGADDRESS
+import ViewResolver   from 0xVIEWRESOLVERADDRESS
 
-pub contract TopShot: NonFungibleToken {
+access(all) contract TopShot: NonFungibleToken, ViewResolver {
```

Comment:
There is no need to add `ViewResolver` here since it is already inherited with `NonFungibleToken`.